web/modules/custom/bnf/tests/src/Unit/Mapper/ParagraphAccordionTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\bnf\Unit\Mapper;
+
+use Drupal\bnf\GraphQL\Operations\GetNode\Node\Paragraphs\AccordionDescription\Text as GraphqlDescriptionText;
+use Drupal\bnf\GraphQL\Operations\GetNode\Node\Paragraphs\AccordionTitle\Text as GraphqlTitleText;
+use Drupal\bnf\GraphQL\Operations\GetNode\Node\Paragraphs\ParagraphAccordion as GraphqlParagraphAccordion;
+use Drupal\bnf\Plugin\bnf_mapper\ParagraphAccordion;
+use Drupal\paragraphs\Entity\Paragraph;
+
+/**
+ * Tests the text_body paragraph mapper.
+ */
+class ParagraphAccordionTest extends EntityMapperTestBase {
+
+  const ENTITY_NAME = 'paragraph';
+  const ENTITY_CLASS = Paragraph::class;
+
+  /**
+   * Test text paragraph mapping.
+   */
+  public function testParagraphTextBodyMapping(): void {
+    $this->storageProphecy->create([
+      'type' => 'accordion',
+    ])->willReturn($this->entityProphecy);
+
+    $mapper = new ParagraphAccordion([], '', [], $this->entityManagerProphecy->reveal());
```

Comment:
Consider adding the mapper `Mapper` suffix to mapper implementations.

I know it is duplicated from the namespace but the current code reads a bit weird. I am a bit conflicted by my suggestion but I guess this is also why it is mentioned by the Drupal code guidelines:

> Classes and interfaces should have names that stand alone to tell what they do without having to refer to the namespace, read well, and are as short as possible without losing functionality information or leading to ambiguity.